tests/test_sourcecode.py

```diff
@@ -107,3 +107,54 @@ def test_cqa_mypy(self):
                     output += '\n\n' + ex.stderr.decode()
                 raise AssertionError(
                     f'mypy validation failed:\n{output}') from None
+
+    def test_clippy(self):
```

Comment:
These 2 tests are failing on nightlies (where we run tests with released binary):

https://github.com/edgedb/edgedb/actions/runs/10481913983/job/29032928532

You can test this by pushing code into a branch in the main repo and manually triggering the `Dry Run` CI.